src/AggregateFunctions/fuzzers/aggregate_function_state_deserialization_fuzzer.cpp

```diff
@@ -12,10 +12,33 @@
 
 #include <Interpreters/Context.h>
 
+#include <Functions/CastOverloadResolver.h>
+
 #include <AggregateFunctions/registerAggregateFunctions.h>
 
 #include <base/scope_guard.h>
 
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int NOT_IMPLEMENTED;
+}
+
+class IFunctionBase;
+using FunctionBasePtr = std::shared_ptr<const IFunctionBase>;
+
+FunctionBasePtr createFunctionBaseCast(
+    ContextPtr, const char *, const ColumnsWithTypeAndName &, const DataTypePtr &, std::optional<CastDiagnostic>, CastType)
+{
+    throw Exception(ErrorCodes::NOT_IMPLEMENTED, "Type conversions are not implemented for Library Bridge");
```

Comment:
But it isn't a library bridge :)

Reply:
will fix, it's just a fast forward to return it to working state